scripts/setup/requirements.nxp.txt

```diff
@@ -3,4 +3,5 @@ jsonschema>=4.17.0
 pycryptodome>=3.20.0
 intelhex
 cbor
-cbor2
\ No newline at end of file
+cbor2
+cmake>=3.30.0
```

Comment:
Do we need cmake as a python module as a dependency?

Other builds use west as well and they do not have a cmake dependency. Why the difference?

Reply:
NXP FreeRTOS SDK needs to be built with a cmake version> 3.30.0 https://mcuxpresso.nxp.com/mcuxsdk/latest/html/gsd/installation.html#basic-tools 
Here we are outside of the Zephyr environment and therefore we don't have Zephyr tools and cmake>=3.30 need to be manually installed. We could add this in our NXP docker img but for users who are not building within a docker img a manual installation step would be required. That is why adding it as part of the bootstrap install remove the necessity of installing it manually.
But let me know if it is not acceptable for you.